backend/generate_cycle_count_data.py

```
@@ -0,0 +1,275 @@
+
+# Add the parent directory to the path so we can import the models
+sys.path.append(os.path.dirname(os.path.dirname(os.path.abspath(__file__))))
+
+# Import the models
+from models import db, Tool, Chemical, User
+from models_cycle_count import (
+    CycleCountSchedule, CycleCountBatch, CycleCountItem,
+    CycleCountResult, CycleCountAdjustment
+)
+
+def generate_sample_data():
```

Pylint (3.3.7)

[refactor] Too many local variables (29/15)

(R0914)